pkgs/top-level/python-packages.nix

```diff
@@ -19253,6 +19253,22 @@ EOF
     };
   };
 
+  vxi11 = buildPythonPackage rec {
```

Comment:
Python packages should have a dedicated directory in `pkgs/development/python-modules` and be included using `callPackage`.